docs/document-level-security/SERVICENOW.md

```diff
@@ -0,0 +1,23 @@
+### Setting up the ServiceNow connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+## Document level security
+
+Document level security (DLS) enables you to restrict access to documents based on a user's permissions. This feature is available by default for the ServiceNow connector.
+
+Refer to [document level security](https://www.elastic.co/guide/en/enterprise-search/master/dls.html) for more information.
+
+**Note:** Refer to [DLS in Search Applications](https://www.elastic.co/guide/en/enterprise-search/master/dls-e2e-guide.html) to learn how to ingest data from ServiceNow with DLS enabled, when building a search application.
+
+#### Additional Configuration
+
+##### `Enable document level security`
+
+Toggle to enable [document level security (DLS)](https://www.elastic.co/guide/en/enterprise-search/master/dls.html). When enabled:
+- Full syncs will fetch access control lists for each document and store them in the `_allow_access_control` field.
+- Access control syncs will fetch users' access control lists and store them in a separate index.
+
+#### Limitations
+
+- ServiceNow connector does not support ServiceNow scripted and conditional permissions.
```

Comment:
Should we document what we do support? Sharing the mapping of roles to resources that we're using?

Reply:
sure, added